dev_guide/managing_images.adoc

```diff
@@ -111,7 +111,7 @@ xref:../install_config/install/docker_registry.adoc#registry-known-issues[known
 issue]
 endif::[]
 ifdef::openshift-online,openshift-dedicated,atomic-registry[]
-https://docs.openshift.com/enterprise/3.2/release_notes/ose_3_2_release_notes.html#ose-32-known-issues[known issue]
+https://docs.openshift.com/enterprise/3.2/release_notes/ose_3_2_release_notes.adoc#ose-32-known-issues[known issue]
```

Comment:
@vikram-redhat Not that it matters here since this is the enterprise-3.2 repo, but in this case this is link to an external URL, so shoulda stayed `.html`.